SDWebImage/Core/SDAnimatedImage.m

```diff
@@ -182,6 +182,8 @@ - (instancetype)initWithData:(NSData *)data scale:(CGFloat)scale options:(SDImag
 #else
         self = [super initWithCGImage:image.CGImage scale:MAX(scale, 1) orientation:image.imageOrientation];
 #endif
+        // Defines the associated object that holds the format for static images
+        super.sd_imageFormat = format;
```

Comment:
I don't think `SDAnimatedImage.sd_imageFormat` is defined behavior ?

Maybe it's better to use `animatedImageFormat` instead

Reply:
Or, it's better to just to override the `sd_imageFormat` and return the `animatedImageFormat`